DashboardPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const DashboardPage = () => {
+  return <h1>Welcome to the dashboard!</h1>;
+};
```

Comment:
This component is currently very simple. As the application grows, consider breaking down this component into smaller, more manageable components.